src/scripts/utils/sanitization.js

```diff
@@ -5,6 +5,7 @@ import DOMPurify from 'dompurify';
  *
  * @param {string} string  - The text to be purified.
  * @param {Object} options - https://github.com/cure53/DOMPurify#can-i-configure-dompurify
+ * @return {{ __html: string }} The sanitized string.
```

Comment:
This throws a type error, the return type of function signature is to fit the `dangerouslySetInnerHTML` prop, it expects `__html` to be a string, but the return type of `DOMPurify.sanitize` is `TrustedHTML` which won't type assert into `string`.